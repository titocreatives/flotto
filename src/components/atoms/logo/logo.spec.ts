import { shallowMount } from "@vue/test-utils";
import Logo from "./";

describe("Atom Logo", () => {
  it("matches snapshot", () => {
    const cmp = shallowMount(Logo);

    expect(cmp.element).toMatchSnapshot();
  });
});
