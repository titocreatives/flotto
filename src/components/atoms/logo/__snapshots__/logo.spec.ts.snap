// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Atom Logo matches snapshot 1`] = `
<div
  class="a logo f-logo"
>
  <h1
    class="logo-text"
    data-text="Flotto"
  >
    Flotto
  </h1>
</div>
`;
